README_LANG.md

```diff
@@ -0,0 +1,126 @@
+
+## SW360 Multi-Language System
+
+### Introduction
+
+SW360 multi-language can support multiple languages. In this way, SW360 can internationalize. With each localization, user can select the appropriate language and contribute more local translations.
+
+### Language Extension 
+
+By the default SW360 language is English, language extensions are added and based on English.
+
+With each language, it will have a special language extension that is language for local.
+
+Language extension contains a pair: key and value
+* Key: used to get value of key and replace text to key in source code.
+* Value: the local language translation.
+
+### How to deploy
+
+Refer to the document:
+
+https://github.com/eclipse/sw360/wiki/Deploy-Upgrade-to-Liferay-7.2
```

Comment:
maybe consider the general (no existing installation in place) page fot it: https://github.com/eclipse/sw360/wiki/Deploy-Natively